app/src/main/java/org/alexcode/snake/Level1.java

```diff
@@ -311,7 +318,7 @@ protected void onPause() {
 
     public boolean onKeyDown(int keyCode, KeyEvent event) {
         if (keyCode == KeyEvent.KEYCODE_BACK) {
-            playGroundView.pause();
+            onPause();
```

Comment:
why did you change this?

Reply:
I wanted to be more clear that I used onPause method